cohorts/views.py

```diff
@@ -1555,52 +1566,62 @@ def write(self, value):
 
 def streaming_csv_view(request, cohort_id=0):
```

Comment:
tbh there's a lot of that in various spots; I suspect it was just someone's coding style